ivy_tests/test_ivy/test_frontends/test_numpy/test_ndarray/test_ndarray.py

```diff
@@ -3638,6 +3641,52 @@ def test_numpy_ndarray_transpose(
     )
 
 
+# var
+@handle_frontend_method(
+    class_tree=CLASS_TREE,
+    init_tree="numpy.array",
+    method_name="var",
+    dtype_x_axis=_statistical_dtype_values(function="var"),
+    dtype=helpers.get_dtypes("valid", full=False, none=True),
+    where=np_frontend_helpers.where(),
+    keep_dims=st.booleans(),
```

Comment:
```suggestion
    keepdims=st.booleans(),
```
https://github.com/unifyai/ivy/actions/runs/6215411246/job/17087315225?pr=22336#step:3:4514 - This is the the reason for this error on the CI. Let's update this and test again ! :)